### PR TITLE
Fix missing images in tutorial

### PR DESCRIPTION
**Pull request by ledsoft**

### CODE REVIEW

File: _pages/tutorial.markdown [hunk 1]
```diff
@@ -27,12 +27,12 @@ Advanced tutorial and documentation describing all tools in TermIt is under cons
 ## Sandbox
 The page <a href="https://kbss.felk.cvut.cz/termit-demo">https://kbss.felk.cvut.cz/termit-demo</a> contains a live demo version of TermIt. It allows various levels of access based on the user permission:
 
-* No account -- it is possible only to browse data, it is not possible to edit term or vocabularies, show documents or other sources. Functionality is very limited. This Regime shall serve to public to view the data.
+* No account -- it is possible only to browse data, it is not possible to edit term or vocabularies, show documents or other sources. Functionality is very limited. This Regime shall serve to the public to view the data.
```
Comment (psiotwo) [inline _pages/tutorial.markdown:30]: Consider aligning the terminology - permissions vs. roles vs. accounts in this paragraph are introduced in a confusing manner.


Below, you are speaking about roles, so e.g. if 'role' is picked as the correct term, the wording would change as follows:

... based on the user role:

* Guest - ...
* Reader - ...
* Editor - ...
* Administrator - ...

**Review by psiotwo — CHANGES_REQUESTED** [review]
One confusing point, see above, otherwise LGTM

File: _pages/tutorial.markdown (hunk 1 above)
Comment (ledsoft) [reply _pages/tutorial.markdown]: Good point, fixed.

**Review by psiotwo — APPROVED** [review]
LGTM

**Pull request merged**